src/screens/Home/Home.js

```diff
@@ -756,6 +756,12 @@ class HomeScreen extends React.Component<Props, State> {
                 onPress={() => navigation.navigate(MANAGE_DETAILS_SESSIONS)}
                 wrapperPaddingHorizontal={spacing.mediumLarge}
               />
+              {/* <SettingsListItem */}
```

Comment:
Probably was used for debug purpose? Maybe not needed anymore?

Reply:
Updated in d685df4